### PR TITLE
Endoc 414-Add Development Concepts page to next and v6.3.2

### PR DESCRIPTION
**Pull request by jyunmitch**
This is a draft and work in progress. We can play with page title, links and everything else.

I tried to add some of the concepts-overview content but it did not flow right so kept it separate for now. It made more sense to keep this a development process concepts page. Further discussion is needed, especially for the navigation sidebar links under docs/getting-started/.

I think the conflicts are the getting started nav list with the solutions template landing page and I've renamed some of the files in that section

### CODE REVIEW

File: vuepress/docs/next/docs/getting-started/development-concepts.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+sidebarDepth: 2
+---
+
+# Development Concepts on Entando
+ 
+On Entando Composition Platform, the development of an application is broken down into modular steps of create, curate, compose, and consume. In the simplest terms, you start by **creating** functional building blocks, **curate** and organize the blocks in a catalogue, **compose** them on a page or website, which is then **consumed** in an app where it can be scaled and improved upon.
```
Comment (nshaw) [inline vuepress/docs/next/docs/getting-started/development-concepts.md:7]: "On Entando Composition Platform" -> "With the Entando Platform,"
"modular steps" -> "distinct steps" or "discrete steps" Not sure there.
"catalogue" -> "catalog".  Let's use American English here.

File: vuepress/docs/next/docs/getting-started/development-concepts.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+# Development Concepts on Entando
+ 
+On Entando Composition Platform, the development of an application is broken down into modular steps of create, curate, compose, and consume. In the simplest terms, you start by **creating** functional building blocks, **curate** and organize the blocks in a catalogue, **compose** them on a page or website, which is then **consumed** in an app where it can be scaled and improved upon. 
+
+Entando provides a composable environment where these building blocks can be built independently, enabling parallel teams to accelerate development and innovation on a cloud and Kubernetes-native stack.
+
+![Entando Platform Development Process](./img/concepts.png)
+
+## Create Components 
+The first step in building a modern application on Entando is to design and create each independant component or functional block for your application. Developers can use Entando Component Generator or Blueprint to automate the building and packaging of project files, with the flexibility to customize. In this way, new business capabilities are integrated into your application without overhauling existing systems.
```
Comment (nshaw) [inline vuepress/docs/next/docs/getting-started/development-concepts.md:14]: "the" Entando Component Generator
"are integrated" -> "can be integrated"
"overhauling" -> "rewriting"

File: vuepress/docs/next/docs/getting-started/development-concepts.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Entando provides a composable environment where these building blocks can be built independently, enabling parallel teams to accelerate development and innovation on a cloud and Kubernetes-native stack.
+
+![Entando Platform Development Process](./img/concepts.png)
+
+## Create Components 
+The first step in building a modern application on Entando is to design and create each independant component or functional block for your application. Developers can use Entando Component Generator or Blueprint to automate the building and packaging of project files, with the flexibility to customize. In this way, new business capabilities are integrated into your application without overhauling existing systems. 
+
+For more information on how to Create Components on Entando, see the docs below or follow a tutorial:
```
Comment (nshaw) [inline vuepress/docs/next/docs/getting-started/development-concepts.md:16]: "see the docs and tutorials below"

File: vuepress/docs/next/docs/getting-started/development-concepts.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+| Documentation | Tutorials        |
+| :------------ | :--------- |
+| [Entando Component Generator](../create/component-gen-overview.md) | [Micro Frontends](../../tutorials/create/mfe/) |
+| [Entando Blueprint Feature](../create/blueprint-features.md) | [Microservices](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md) |
+| [Component Generation Technologies](../create/component-gen-tech.md) | [Build and Publish Project Bundles](../../tutorials/create/pb/publish-project-bundle.md) |
+
+
+	
+## Curate and Share Components 
+On the Entando Platform, Curate and Share means developers manage a central catalog of components and share it across an organization. Entando's distributed and composable development process allows you to quickly edit, test, analyze, and publish a singlular component or packaged business capability (PBC) repeatedly, without touching other parts of your application. This includes the ability to guide content, versioning, and metadata through the use of bundles.
```
Comment (nshaw) [inline vuepress/docs/next/docs/getting-started/development-concepts.md:27]: "developers manage a central catalog.." -> "developer teams can share components within a central catalog made available to multiple teams or organizations"
"singlular" -> singular
"guide content..." -> "publish content and pages along with functional components such as micro frontends and microservices."

File: vuepress/docs/next/docs/getting-started/development-concepts.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+| [Entando Component Generator](../create/component-gen-overview.md) | [Micro Frontends](../../tutorials/create/mfe/) |
+| [Entando Blueprint Feature](../create/blueprint-features.md) | [Microservices](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md) |
+| [Component Generation Technologies](../create/component-gen-tech.md) | [Build and Publish Project Bundles](../../tutorials/create/pb/publish-project-bundle.md) |
+
+
+	
+## Curate and Share Components 
+On the Entando Platform, Curate and Share means developers manage a central catalog of components and share it across an organization. Entando's distributed and composable development process allows you to quickly edit, test, analyze, and publish a singlular component or packaged business capability (PBC) repeatedly, without touching other parts of your application. This includes the ability to guide content, versioning, and metadata through the use of bundles. 
+
+To Curate and Share on Entando, see the docs below or follow a tutorial:
```
Comment (nshaw) [inline vuepress/docs/next/docs/getting-started/development-concepts.md:29]: "docs or tutorials below:"

File: vuepress/docs/next/docs/getting-started/development-concepts.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+| :------------ | :-------- |
+| [Bundle and Component Descriptors](../curate/ecr-bundle-details.md) | [Install Bundles from a Private Git Repository](../../tutorials/curate/ecr-private-git-repo.md) |
+| [Filtering Bundles](../curate/ecr-bundle-filters.md) | [Install Bundle Plugins from a Private Image Repo](../../tutorials/curate/ecr-private-images.md) |
+| [Customize Bundle Info in App Builder](../curate/ecr-bundle-presentation-config.md) |   |
+| [Bundle Version and Updates FAQ](../curate/ecr-bundle-versions-faq.md) |  |
+| [Bundle Uninstall](../curate/ecr-uninstall-flow.md) |  |
+
+
+## Compose an Application 
+Business IT groups and developers Compose an Application with the Entando App Builder, a low-code composition UI. Assemble or update an application from the components already built in step one, adding templates, widgets, or full PBCs for functionality. The Entando Component Manager and Repository are built in to the App Builder, making the composable process flexible and agile by managing all your components there.
```
Comment (nshaw) [inline vuepress/docs/next/docs/getting-started/development-concepts.md:40]: Might need to format "Compose an Application" and do the same in the other sections

Comment (nshaw) [reply vuepress/docs/next/docs/getting-started/development-concepts.md:40]: "built in to the App Builder" -> "integrated with the App Builder"

File: vuepress/docs/next/docs/getting-started/development-concepts.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+| :------------ | :-------- |
+| [Welcome Wizard](../compose/welcome-wizard.md) | [Page Management](../../tutorials/compose/page-management.md) |
+| [Entando Component Repository](../compose/ecr-overview.md) | [Widgets and Fragments](../../tutorials/compose/widgets-fragments.md) |
+| [Entando App Builder]<!--(../compose/app-builder.md)--> | [Content Creation](../../tutorials/compose/content-tutorial.md) |
+| [Entando Component Manager]<!--(../compose/ecm-overview.md)--> | [Content Templates](../../tutorials/compose/content-templates-tutorial.md)  |
+|  | [Digital Assets](../../tutorials/compose/digital-assets-tutorial.md) |
+
+## Consume an Application 
+When the end-user interacts with an application, the Consume step provides an opportunity to monitor, analyze and enhance each building block. On Entando, the modular capability allows organizations to scale the app accordingly. This is the last step in the continuous process of composable apps, where feedback to creators and composers is built in. This means adding or editing web content, changing engine rules, or managing processes and useage are done independently and upgrades are responsive.
```
Comment (nshaw) [inline vuepress/docs/next/docs/getting-started/development-concepts.md:52]: useage -> usage

Comment (nshaw) [reply vuepress/docs/next/docs/getting-started/development-concepts.md:52]: This paragraph was awkward in the source doc as well and isn't even technically accurate. We don't have a feature "where feedback to creators and composers is built in".  Maybe, "The 'Consume' step is the runtime phase of an Entando Application where an end-user (or consumer) interacts with the application to complete their specific task.  The modular design of the platform allows the application and services to be scaled according to that use.  In the same way the application can be continuously improved by adding or updating web content, changing engine roles, and deploying seamless upgrades to individual components.

File: vuepress/docs/next/docs/getting-started/development-concepts.md (hunk 5 above)
Comment (nshaw) [reply vuepress/docs/next/docs/getting-started/development-concepts.md:29]: Maybe we don't need this line for the first 3 Cs? We don't have it for Consume and I think it reads fine.

File: vuepress/docs/v6.3.2/docs/create/blueprint-features.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
+# Entando Blueprint Features
```
Comment (nshaw) [inline vuepress/docs/v6.3.2/docs/create/blueprint-features.md:1]: I'm confused. Was this added to 6.3.2 just as part of this PR? Was trying to make sense of the diff.

**Review by nshaw — CHANGES_REQUESTED** [review]
Might need a new PR with just the differences for ENDOC-414. This one seems to have commits from the restructure as well...

File: vuepress/docs/next/docs/getting-started/development-concepts.md (hunk 6 above)
Comment (jyunmitch) [reply vuepress/docs/next/docs/getting-started/development-concepts.md:40]: what did you mean by format "Compose an Application"?

**Comment by nshaw** [comment]
Closing in favor of a new PR

**Comment by nshaw** [comment]
Closing in favor of a new PR

File: vuepress/docs/next/docs/getting-started/development-concepts.md (hunk 6 above)
Comment (nshaw) [reply vuepress/docs/next/docs/getting-started/development-concepts.md:40]: @jyunmitch , sorry, missed this comment. I meant to add some formatting on the phrase in the text, bold or italics or backticks, whatever matches the style guide.

**Pull request closed without merging**